public/app/plugins/datasource/influxdb/response_parser.ts

```diff
@@ -12,17 +12,27 @@ export default class ResponseParser {
       return [];
     }
 
-    var series = influxResults.series[0];
-    return _.map(series.values, (value) => {
-      if (_.isArray(value)) {
-        if (query.toLowerCase().indexOf('show tag values') >= 0) {
-          return { text: (value[1] || value[0]) };
+    var res = [];
+    _.each(influxResults.series, (serie) => {
+      _.each(serie.values, (value) => {
+        if (_.isArray(value)) {
+          if (query.toLowerCase().indexOf('show tag values') >= 0) {
```

Comment:
Minor performance thing here, try to have this check outside the loop once, instead of on every loop